.github/release-drafter.yml

```diff
@@ -0,0 +1,14 @@
+name-template: v$NEXT_PATCH_VERSION
+tag-template: v$NEXT_PATCH_VERSION
+categories:
+  - title: 🚀 Improvements
+    label: enhancement
+  - title: 🐛 Bug Fixes
+    label: bug
```

Comment:
It should be nice to also add a "feature" category? What do you think? :D

Reply:
Added `feature` label from PrestaShop/PrestaShop, then updated the release drafter accordingly